__std__/nix/toolchain/library/cabal-project-index-state.nix

```diff
@@ -1,4 +1,3 @@
 { inputs, cell }:
 
-# TODO(std) revisit why we need this, and why we were parsing it from the actual cabal.project
-"2022-08-31T00:00:00Z"
+inputs.cells.plutus.library.plutus-project.index-state
```

Comment:
please leave the todo, I'm not sure if this is actually equivalent to what we were doing before